app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java

```diff
@@ -0,0 +1,249 @@
+package com.cornellappdev.android.eatery;
+
+import android.content.Intent;
+import android.graphics.drawable.GradientDrawable;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.SearchView;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AutoCompleteTextView;
+import android.widget.Button;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.cornellappdev.android.eatery.model.EateryBaseModel;
+import com.cornellappdev.android.eatery.model.enums.CampusArea;
+import com.cornellappdev.android.eatery.model.enums.PaymentMethod;
+import com.cornellappdev.android.eatery.presenter.MainListPresenter;
+import com.cornellappdev.android.eatery.presenter.MainPresenter;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.cornellappdev.android.eatery.MainActivity.PAYMENT_CARD;
+import static com.cornellappdev.android.eatery.MainActivity.PAYMENT_SWIPE;
+import static com.cornellappdev.android.eatery.model.enums.CampusArea.CENTRAL;
+import static com.cornellappdev.android.eatery.model.enums.CampusArea.NORTH;
+import static com.cornellappdev.android.eatery.model.enums.CampusArea.WEST;
+
+public class MainListFragment extends Fragment
+		implements MainListAdapter.ListAdapterOnClickHandler, View.OnClickListener {
+	private MainListPresenter presenter;
+	private MainPresenter mPresenter;
+	private RecyclerView mRecyclerView;
+	private MainListAdapter listAdapter;
+	private Button northButton;
+	private Button westButton;
+	private Button centralButton;
+	private Button swipesButton;
+	private Button brbButton;
+	private Set<Button> areaButtonsPressed;
+	private Set<Button> paymentButtonsPressed;
```

Comment:
shouldn't all the variables declared from `45` to `52` be prefaced with `m`? https://source.android.com/setup/contribute/code-style#follow-field-naming-conventions